views/partials/rss/reports/search.jade

```diff
@@ -0,0 +1,36 @@
+ui-view
+  .row
+    .col-xs-12(ng-switch="searchVM.list.status")
+      div(ng-switch-when="LOADED", ng-switch="searchVM.list.length")
+        div(ng-switch-when="0")
+          .alert.alert-info.text-center No reports found.
+        div(ng-switch-default)
+          .panel-group.panel-group-report
+            .panel.panel-default.z-depth-1(ng-repeat="report in searchVM.list")
+              .panel-heading
+                .h4
+                  a.btn-block(data-toggle="collapse" data-parent=".panel-group-report" data-target="#report-{{ $index }}")
+                    span.item-text {{ report.timestamp | amDateFormat: 'ddd, MMM Do YYYY, HH:mm' }}
+                  small
+                    a(ui-sref="rss.models.update({ productClass: report.productClass })" ui-sref-opts="{ inherit: false }") {{ report.productClass }}
+              .panel-collapse.collapse(id="report-{{ $index }}")
+                .panel-body
```

Comment:
I prefer this panel to be distributed in columns. Otherwise, it seems very chaotic having text over text in a single column